scripts/us_bjs/nps/import_data.py

```diff
@@ -65,13 +79,21 @@ def save_csv(df, filename):
     df.to_csv(filename + '.csv', index=False)
 
 
+def get_input_file():
+    os.makedirs(input_files, exist_ok=True)
+    file_util.file_copy(f'{FLAGS.gs_path}{FLAGS.input_file}',
+                        f'{input_files}/{FLAGS.input_file}')
+
+
 def main(args):
-    df = pd.read_csv(FLAGS.input_file, delimiter='\t')
-    processed_df = preprocess_df(df)
+    INPUT_FILE_PATH = os.path.join(input_files, FLAGS.input_file)
+    get_input_file()
```

Comment:
Can we check the failure conditions from function  calls to properly log the error ? Also add the block in try catch with exception block for specific errors like FileNotFoundError etc.